packages/navlink/src/types.ts

```diff
@@ -0,0 +1,15 @@
+import { IconSize } from '@fiscozen/icons/src/types'
+import { RouteLocation } from 'vue-router'
+
+export interface FzNavlinkProps<T = void> {
+  iconName?: string
+  iconSize?: IconSize
+  label?: string
+  disabled?: boolean
+  meta?: T
+}
+
+type PartialExcept<T, K extends keyof T> = Pick<Required<T>, K> & Partial<T>
+
+export type CustomRouteLocation = PartialExcept<RouteLocation, 'path' | 'name'>
+export interface FzRouterNavlinkProps extends FzNavlinkProps<CustomRouteLocation> {}
```

Comment:
avrei fatto qualcosa tipo
`type FzRouterNavlinkProps = FzNavlinkProps & { to: string | RouteLocation }`
in questo modo `meta` si può levare da FzNavlinkProps, dato che mi sembra non serva a nulla in `FzNavlink`, mentre l'utilizzo di `FzRouterNavlink` diventa più intuitivo perché riprende `router-link`

Reply:
non sono d'accordo, il meta è facilmente utile anche in qualche navlink dove la navigazione non sia primaria e quindi sarà necessario controllare metadati sull'elemento.
Preferirei poi quindi mantenere coerenza tra i due componenti znavlink e fzrouternavlink e quindi usare lo stesso campo.